include/magma/vk/textures/TextureManager.h

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file TextureManager.h
+ * @brief Create and control textures
+ * @version 0.1
+ * @date 2021-02-12
+ */
+#pragma once
+#include <string>
+#include <unordered_map>
+
+#include "magma/vk/commandBuffer.h"
+#include "magma/vk/textures/Texture.h"
+
+class LogicalDevice;
+
+class TextureManager {
+public:
+    TextureManager(LogicalDevice &device);
+    ~TextureManager();
+
+    bool textureExist(const std::string &name) const;
+    Texture &getTexture(const std::string &name);
+
+    Texture &createTexture2D(const std::string &name, VkFormat format, VkExtent3D extent,
+        VkImageUsageFlags usage, VkImageAspectFlags aspectMask);
+
+    void setLayout(Texture &texture, VkImageLayout newLayout);
+    void copyFromBuffer(Texture &texture, VkBuffer buffer);
```

Comment:
potentially there will be the same invert function, which copies from texture to buffer.
So this function should better be called
```
copyBufferToTexture
```
or
```
copyBufToTex
```

Reply:
Done